packages/email-components/src/components/Link/README.md

```diff
@@ -8,13 +8,14 @@ The `mjc-link` component to create link.
 
 ## Available props
 
-| Name             | Type                               | Default   | Required | Description                                                      |
-| ---------------- | ---------------------------------- | --------- | -------- | ---------------------------------------------------------------- |
-| `align`          | `left \| center \| right`          | `left`    | no       | Text aligment                                                    |
-| `color`          | `primary \| secondary \| inverted` | `primary` | no       | Link color                                                       |
-| `href`           | `string`                           | -         | yes      | Link to                                                          |
-| `padding-bottom` | `unit(px,%)`                       | -         | no       | Bottom spacing                                                   |
-| `padding-top`    | `unit(px,%)`                       | -         | no       | Top spacing                                                      |
-| `padding`        | `unit(px){1,4}`                    | -         | no       | Spacing                                                          |
-| `rel`            | `string`                           | -         | no       | Relationship between a linked resource and the current document. |
-| `size`           | `small \| medium`                  | `medium`  | no       | Size of the text                                                 |
+| Name             | Type                                 | Default     | Required | Description                                                      |
+| ---------------- | ------------------------------------ | ----------- | -------- | ---------------------------------------------------------------- |
+| `align`          | `left \| center \| right`            | `left`      | no       | Text aligment                                                    |
+| `color`          | `primary \| secondary \| inverted`   | `primary`   | no       | Link color                                                       |
+| `href`           | `string`                             | -           | yes      | Link to                                                          |
+| `padding-bottom` | `unit(px,%)`                         | -           | no       | Bottom spacing                                                   |
+| `padding-top`    | `unit(px,%)`                         | -           | no       | Top spacing                                                      |
+| `padding`        | `unit(px){1,4}`                      | -           | no       | Spacing                                                          |
+| `rel`            | `string`                             | -           | no       | Relationship between a linked resource and the current document. |
```

Comment:
It is not a sentence, since the predicate is missing. The dot at the end of line is not supposed to be there. Please remove it.